src/main/java/seedu/address/logic/parser/RemindEventCommandParser.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.*;
+
+import java.util.stream.Stream;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.AddContactCommand;
+import seedu.address.logic.commands.RemindEventCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+
+/**
+ * Parses input arguments and creates a new AddContactCommand object
+ */
+public class RemindEventCommandParser implements Parser<RemindEventCommand> {
+
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddContactCommand
+     * and returns an AddContactCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
AddContactCommand references should not be here. Please double check if copy pasting has left anything that is not relevant and needs to be changed.